Crypto/PublicKey+Windows.cc

```diff
@@ -168,6 +168,46 @@ namespace litecore::crypto {
         return winCert;
     }
 
+    PCCERT_CONTEXT getWinCert(PublicKey *subjectKey) {
+        auto keyData = subjectKey->publicKeyData(KeyFormat::Raw);
+        unsigned char hash[16];
+
+        mbedtls_md5_context context;
+        mbedtls_md5_init(&context);
+        mbedtls_md5_starts(&context);
+        mbedtls_md5_update(&context, (const unsigned char *)keyData.buf, keyData.size);
+        mbedtls_md5_finish(&context, hash);
+        mbedtls_md5_free(&context);
+
+        CRYPT_HASH_BLOB hashBlob {
+                16,
+                hash
+        };
+
+        auto* const store = CertOpenStore(
+                CERT_STORE_PROV_SYSTEM_A,
+                X509_ASN_ENCODING,
+                NULL,
+                CERT_SYSTEM_STORE_CURRENT_USER,
+                "CA"
+        );
+
+        auto winCert = CertFindCertificateInStore(
+                store,
+                X509_ASN_ENCODING,
+                0,
+                CERT_FIND_PUBKEY_MD5_HASH,
+                &hashBlob,
+                nullptr
+        );
+
+        DEFER {
```

Comment:
There is no need to use DEFER here since there is nothing left in the method.  You can simply call `CertCloseStore` without the DEFER.  DEFER is there to act sort of like a C++ destructor, so that no matter what happens after this call is guaranteed to happen.